applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_internal.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BROADCAST_INTERNAL_H_
+#define _BROADCAST_INTERNAL_H_
+
+#include "le_audio.h"
+
+#if CONFIG_BT_AUDIO_BROADCAST_CONFIGURABLE
+#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_BAP_LC3_PRESET_CONFIGURABLE(                                                            \
+		BT_AUDIO_LOCATION_FRONT_LEFT | BT_AUDIO_LOCATION_FRONT_RIGHT,                      \
+		BT_AUDIO_CONTEXT_TYPE_MEDIA, CONFIG_BT_AUDIO_BITRATE_BROADCAST_SRC)
+
+#elif CONFIG_BT_BAP_BROADCAST_16_2_1
+#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_BAP_LC3_BROADCAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT |                          \
+						   BT_AUDIO_LOCATION_FRONT_RIGHT,                  \
+					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_BAP_BROADCAST_24_2_1
+#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_BAP_LC3_BROADCAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT |                          \
+						   BT_AUDIO_LOCATION_FRONT_RIGHT,                  \
+					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_BAP_BROADCAST_16_2_2
+#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_BAP_LC3_BROADCAST_PRESET_16_2_2(BT_AUDIO_LOCATION_FRONT_LEFT |                          \
+						   BT_AUDIO_LOCATION_FRONT_RIGHT,                  \
+					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_BAP_BROADCAST_24_2_2
+#define BT_BAP_LC3_BROADCAST_PRESET_NRF5340_AUDIO                                                  \
+	BT_BAP_LC3_BROADCAST_PRESET_24_2_2(BT_AUDIO_LOCATION_FRONT_LEFT |                          \
+						   BT_AUDIO_LOCATION_FRONT_RIGHT,                  \
+					   BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#else
+#error Unsupported LC3 codec preset for broadcast
+#endif /* CONFIG_BT_AUDIO_BROADCAST_CONFIGURABLE */
+
+/**
+ * @brief	Get bt_data containing the data to advertise.
+ *
+ * @param[out]	ext_adv		Pointer to the pointer of bt_data to advertise extended.
+ * @param[out]	ext_adv_size	Pointer to size of @p ext_adv.
+ * @param[out]	per_adv		Pointer to the pointer of bt_data to advertise periodically.
+ * @param[out]	per_adv_size	Pointer to size of @p per_adv.
+ */
+void broadcaster_adv_get(const struct bt_data **ext_adv, size_t *ext_adv_size,
+			 const struct bt_data **per_adv, size_t *per_adv_size);
+
+/**
+ * @brief	Start the Bluetooth LE Audio stream.
```

Comment:
```suggestion
 * @brief	Start the Bluetooth LE Audio broadcaster.
```

Same for the functions below

Reply:
Done